superset/migrations/versions/18532d70ab98_fix_table_unique_constraint_in_mysql.py

```diff
@@ -27,16 +27,20 @@
 down_revision = "3fbbc6e8d654"
 
 from alembic import op
+from sqlalchemy.dialects.mysql.base import MySQLDialect
 
 
 def upgrade():
-    try:
+    bind = op.get_bind()
+    if isinstance(bind.dialect, MySQLDialect):
         # index only exists in mysql db
         with op.get_context().autocommit_block():
             op.drop_constraint("table_name", "tables", type_="unique")
-    except Exception as ex:
-        print(ex)
 
 
 def downgrade():
-    pass
+    bind = op.get_bind()
+    if isinstance(bind.dialect, MySQLDialect):
+        # index only exists in mysql db
+        with op.get_context().autocommit_block():
+            op.add_constraint("table_name", "tables", type_="unique")
```

Comment:
AttributeError: module 'alembic.op' has no attribute 'add_constraint'

I would say - let's pass on the downgrade. That constraint was never needed